grakn-client/src/main/java/ai/grakn/client/BatchExecutorClient.java

```diff
@@ -178,7 +180,7 @@ public static Builder newBuilder() {
         return new Builder();
     }
 
-    //TODO: Remove this method used only by docs tests
+    @API
```

Comment:
I'd deprecate this (or keep the TODO), since we wanted to remove it but it's used by the doc tests.

Reply:
This is actually the way we want users to interface with the BEC programatically. So it's actual public API . . . according to the docs at least.